packages/react/src/primitives/HighlightMatch/HighlightMatch.tsx

```diff
@@ -12,8 +12,9 @@ export const HighlightMatchPrimitive: Primitive<HighlightMatchProps, 'span'> = (
   ref
 ) => {
   const matchTestId = getTestId(testId, 'match');
-  const startIndex =
-    children?.toLocaleLowerCase().indexOf(query?.toLocaleLowerCase()) ?? -1;
+  const startIndex = children
+    ?.toLocaleLowerCase()
+    .indexOf(query?.toLocaleLowerCase());
```

Comment:
I do not know why I add this nullish, `indexOf` will return -1 if cannot find anything